example/test_problem/Template/Input__Parameter

```diff
@@ -380,6 +380,7 @@ SPEC_INT_GHOST_BOUNDARY       4           # ghost boundary size for spectral int
 OPT__OUTPUT_TOTAL             1           # output the simulation snapshot: (0=off, 1=HDF5, 2=C-binary) [1]
 OPT__OUTPUT_PART              0           # output a single line or slice: (0=off, 1=xy, 2=yz, 3=xz, 4=x, 5=y, 6=z, 7=diag) [0]
 OPT__OUTPUT_TEXT_FORMAT_FLT   %24.16e     # string format of output text files [%24.16e]
+OPT__OUTPUT_TEXT_LENGTH_INT   12          # string length of integer output text files [12]
```

Comment:
- For `OPT__OUTPUT_TEXT_FORMAT_FLT`, change the comment to "string format of floating-point variables in output text files.
- For `OPT__OUTPUT_TEXT_LENGTH_INT`, change the comment to "string length of integer variables in output text files"